TypeScript/TS/tscript.ts

```diff
@@ -0,0 +1,382 @@
+var buttonText:string|number, tempVarTrigo:string|number;
```

Comment:
Format the code with proper spacing.
`var buttonText: string | number;`.

You can use prettier or vscode built-in code formatter.